test11.js

```diff
@@ -0,0 +1,10 @@
+var foo = '' ;
+
+if(foo  === 'bar') {
+  console.log( 'hey' )
+}
+
+
+if(foca  === 'foca') {
+  console.log( 'hey foca' );
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26955)
```suggestion
  console.log( "hey foca" );
```